core/playwright.config.ts

```diff
@@ -63,13 +63,16 @@ const config: PlaywrightTestConfig = {
   /* Fail the build on CI if you accidentally left test.only in the source code. */
   forbidOnly: !!process.env.CI,
   /* Fail fast on CI */
```

Comment:
i think this comment needs to be removed?

Reply:
Good catch! Addressed in https://github.com/ionic-team/ionic-framework/pull/27973